scripts/check_site_changes.sh

```diff
@@ -0,0 +1,7 @@
+# Name: check_site_changes.sh
```

Comment:
It'd be better to prepend the following line:

```sh
#!/usr/bin/env bash
```

This ensures that the shell script is evaluated in Bash when it's run directly.

Reply:
Nice suggestion. Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/commit/4442ca2fcb661e51cf808fd0ba91aa9fd7d39059>.